src/caja-application.c

```diff
@@ -634,44 +634,48 @@ selection_get_cb (GtkWidget          *widget,
 static GtkWidget *
 get_desktop_manager_selection (GdkDisplay *display)
 {
-	/*FIXME: for Wayland we need a new desktop canvas anyway
-    *so when that is ready we will need to add a wayland code
-    *path here to support it
-    */
-    if  (GDK_IS_X11_DISPLAY (display)){
-        char selection_name[32];
-        GdkAtom selection_atom;
-        Window selection_owner;
-        GtkWidget *selection_widget;
-
-        g_snprintf (selection_name, sizeof (selection_name), "_NET_DESKTOP_MANAGER_S0");
-        selection_atom = gdk_atom_intern (selection_name, FALSE);
+    char selection_name[32];
+    GdkAtom selection_atom;
+    Window selection_owner;
+    GtkWidget *selection_widget;
 
+    g_snprintf (selection_name, sizeof (selection_name), "_NET_DESKTOP_MANAGER_S0");
+    selection_atom = gdk_atom_intern (selection_name, FALSE);
+    if  (GDK_IS_X11_DISPLAY (display))
+    {
         selection_owner = XGetSelectionOwner (GDK_DISPLAY_XDISPLAY (display),
                                           gdk_x11_atom_to_xatom_for_display (display,
                                                   selection_atom));
-        if (selection_owner != None)
+
+        if  (selection_owner != None)
```

Comment:
One space after `if`.

Reply:
Found that and removed the extra line too, that entire section should now be unchanged from master after I push this round of edits